roles/grafana/README.md

```diff
@@ -50,7 +50,8 @@ All variables which can be overridden are stored in [defaults/main.yml](defaults
 | `grafana_session` | {} | [session](http://docs.grafana.org/installation/configuration/#session) management configuration section |
 | `grafana_analytics` | {} | Google [analytics](http://docs.grafana.org/installation/configuration/#analytics) configuration section |
 | `grafana_smtp` | {} | [smtp](http://docs.grafana.org/installation/configuration/#smtp) configuration section |
-| `grafana_alerting` | {} | [alerting](http://docs.grafana.org/installation/configuration/#alerting) configuration section |
+| `grafana_alerting` | {} | [alerting](http://docs.grafana.org/installation/configuration/#alerting) configuration section, require Grafana v10 and below |
```

Comment:
This docs show the default value is an empty dict. But it is `{execute_alerts: true}` in the real configuration.

It would be great if we keep it consistent.

The same issue happens on `grafana_unified_alerting`.

Reply:
I have included the default variables in the README now. Alerting is enabled by default for all versions. I believe it makes sense to keep it this way. If you really want to disable alerting, you can simply set the variable in your inventory. Additionally, this change does not break any deployments that are currently using this role or plan to use this role. It will work with both the old and latest versions.